src/serializers/htmlToSlate/index.ts

```diff
@@ -33,24 +33,28 @@ const deserialize = ({
     return null
   }
 
-  const parent = el as Element
-  const isLastChild = index === childrenLength - 1
-  const nodeName = getName(parent)
+  const currentEl = el as Element
```

Comment:
Really appreciate this. I confused myself with this naming! Of course it should be `currentEl` 😅